banking_system/templates/t3_employee_home.html

```diff
@@ -1,5 +1,45 @@
 {% extends 'body.html' %}
 {% block content %}
+<style>
```

Comment:
The side nav could perhaps be done as a template since other use levels will need this